TanmayeeGujar_HW5-2.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
<p>Good point to give a reference, however I asked on your platform, so it would have been good to just check it (eg with types as shown in class)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>